index.html

```diff
@@ -265,6 +265,45 @@ <h3>FindCollabs</h3>
           </p>
         </section>
       </div>
+      <section>
```

Comment:
It's come up that our front page is getting too full, rather than add this there I started a `/resources.md` page but haven't linked it yet as there's not enough content.

Do you want to add this there instead, then we can link up from the home page?

Reply:
I appreciate that markdown is being used to make things easier, but I'm familiar enough with HTML that it's getting in the way of some of the formatting I would like. Regardless, I took a stab at it and I think it's okay, though I would like to use the same buttons as on the main site if at all possible.